### PR TITLE
Split roles for cli & server

### PR DESCRIPTION
**Pull request by myersCody**

### CODE REVIEW

File: install/qpc-tools [hunk 1]
```diff
@@ -14,9 +14,12 @@
 export PATH=$PATH:$ANSIBLE_HOME/bin
 #Note: The playbook path will be sed replaced in the spec file
 # to match the location of where the package is being installed.
-PLAYBOOKFILE="qpc_playbook.yml"
+#PLAYBOOKFILE="qpc_playbook.yml"
```
Comment (abaiken) [inline install/qpc-tools]: You can remove this commented out variable

File: install/qpc-tools [hunk 2]
```diff
@@ -99,25 +102,43 @@ EOM
...
+  ansible-playbook cli/$CLI_PLAYBOOK -vv -K $*
 else
-  echo ansible-playbook $PLAYBOOK -vv $@
-  ansible-playbook $PLAYBOOK -vv $*
+  echo ansible-playbook cli/$CLI_PLAYBOOK -vv $*
+  ansible-playbook cli/$CLI_PLAYBOOK -vv $*
 fi
+if [ $? -eq 0 ]
+then
+  echo "CLI Install complete."
```
Comment (abaiken) [inline install/qpc-tools]: I would change this to "CLI installation complete."

File: install/qpc-tools [hunk 3]
```diff
@@ -99,25 +102,43 @@ EOM
...
+  exit 0
+else
+  echo "CLI Install Failed."
```
Comment (abaiken) [inline install/qpc-tools]: "CLI installation failed. Review the install logs."

File: install/qpc-tools [hunk 4]
```diff
@@ -99,25 +102,43 @@ EOM
...
+  echo ansible-playbook server/$SERVER_PLAYBOOK -vv -K $*
+  ansible-playbook server/$SERVER_PLAYBOOK -vv -K $*
+else
+  echo ansible-playbook server/$SERVER_PLAYBOOK -vv $*
+  ansible-playbook server/$SERVER_PLAYBOOK -vv $*
+fi
 if [ $? -eq 0 ]
 then
-  echo "Installation complete."
+  echo "Quipucords Server Install complete."
```
Comment (abaiken) [inline install/qpc-tools]: "Quipucords server installation complete."  or "Quipucords Server installation complete." (not sure if server should be upper/lowercase for this statement and the one below)

File: install/qpc-tools [hunk 5]
```diff
@@ -99,25 +102,43 @@ EOM
...
   exit 0
 else
-  echo "Installation failed. Review the install logs."
+  echo "Quipucords Server Install Failed."
```
Comment (abaiken) [inline install/qpc-tools]: "Quipucords server installation failed. Review the install logs."

**Review by abaiken — CHANGES_REQUESTED** [review]

**Comment by abaiken** [comment]
This is not working correctly for me. When I run `./qpc-tools install -e isntall_server=false -e install_cli=true` it installs neither and ends in `Quipucords Server Install complete`.

**Review by abaiken — APPROVED** [review]

**Review by kholdaway — CHANGES_REQUESTED** [review]
My PR should be merged

**Review by kholdaway — APPROVED** [review]

**Pull request merged**